.tool-versions

```diff
@@ -1,7 +1,7 @@
 golang 1.20.4
 mockery 2.28.1
 nodejs 16.16.0
-postgres 13.3
+postgres 15.1
```

Comment:
Is this risky? IIRC we usually leave it as the lower supported major version, so we don't accidentally use features introduced in a new one.

Reply:
Sounds good, I can remove